imap.lisp

```diff
@@ -372,12 +379,16 @@
            (ssl-args (cdr server-info))
            ssl port starttls sock)
       (setq ssl (pop-keyword :ssl ssl-args))
-      (setq port (or (pop-keyword :port ssl-args) (server-port ssl server-type)))
+      (setq port (or (pop-keyword :port ssl-args)
+                     (server-port ssl server-type)))
       (setq starttls (pop-keyword :starttls ssl-args))
       (setq sock (socket:make-socket :remote-host server
                                      :remote-port port))
       (when ssl
-        (setq sock (apply #'socket:make-ssl-client-stream sock ssl-args)))
+        (setq sock (apply #'cl+ssl:make-ssl-client-stream
+                          sock
+                          :external-format :iso-8859-1
+                          ssl-args)))
```

Comment:
Thanks for bringing this up! I have applied this to all uses of `make-ssl-client-stream` in https://github.com/orivej/post-office/commit/0ded4606c84ae39512346ef0bb83baf773262fd6. Could you explain the need for the `external-format` argument?